com.unity.shadergraph/Editor/Importers/ShaderSubGraphImporter.cs

```diff
@@ -19,7 +19,7 @@
 namespace UnityEditor.ShaderGraph
 {
     [ExcludeFromPreset]
-    [ScriptedImporter(25, Extension, -905)]
+    [ScriptedImporter(25, Extension, -2001)] // Import before ShaderGraph
```

Comment:
Should this number be smaller than the ShaderGraphImporter ?

Reply:
It could, but inside an import queue (so assets with the same import priority), assets are sorted by dependencies. In the shadergraph importer, the function GatherDependenciesFromSourceFile registers a dependency with subgraphs so they should always be imported before